app/main/views/suppliers.py

```diff
@@ -232,6 +246,8 @@ def list_countersigned_agreement_file(supplier_id, framework_slug):
     agreements_bucket = s3.S3(current_app.config['DM_AGREEMENTS_BUCKET'])
     countersigned_agreement_document = agreements_bucket.get_key(supplier_framework.get('countersignedPath'))
 
+    remove_countersigned_agreement_confirm = request.args.get('remove_countersigned_agreement') == 'True'
```

Comment:
(Nitpicking now) Is it me or is the capital letter here a bit eugh?
```
bool(yaml.safe_load(request.args.get('remove_countersigned_agreement', '')))
```
I think maybe even for suggesting this I should probably go and have a lie down.

Reply:
We do somewhere amongst our legions of functions which call functions which call functions (...) have a "is string yes-sy?" function.

Reply:
Yes-y or truthy? I don't really want to accept any old truthy string - I've been stung by params like `are_you_sure=no` in the past.

I will definitely make `true` lower case though. As you may be able to tell I didn't expend a great deal of thought on this bit :)

Reply:
Yes-y. Truthy would just be `bool(mystr)`.

Reply:
I've used the utils `convert_to_boolean` (I think that's the one you meant?) which should cover the common cases.